app/router.js

```diff
@@ -6,6 +6,17 @@ var Router = Ember.Router.extend({
 });
 
 Router.map(function() {
+  this.route('digidoo', {path: '/'}, function() {
+    this.route('index', function() {
```

Comment:
Why did you add this `index` definition to the `router.js` file? Index routes are automatically registered to the `/` path. Doing this instead has made it so that the homepage no longer shows up.